app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -189,8 +189,9 @@ class Scene extends React.Component {
     // directly on top of each other.
     World.add(
       this.engine.world,
-      Bodies.circle(this.state.width / 2 + (Math.random() - 0.5), -50, 17, {
-        restitution: 0.7,
+      Bodies.circle(this.state.width / 2 + (Math.random() - 0.5), -50, 8, {
+        restitution: 0.25,
+        friction: 0.8,
```

Comment:
`change block properties` is kind of vague. Could you indicate intent or provide some more context in the commit body following the first line?